### PR TITLE
mapserver 7.4.2

### PR DESCRIPTION
**Pull request by Moisan**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by alebcay** [comment]
```
expected replacement of "${PHP5_EXTENSION_DIR}" with #<Pathname:/usr/local/Cellar/mapserver/7.4.0/lib/php/extensions>
```
Source now contains `${PHP_EXTENSION_DIR}` instead of `${PHP5_EXTENSION_DIR}`.

--

Even after the `inreplace` issue is resolved, build still fails. This failure also exists in current version of mapserver (if `--build-from-source` is specified), and this failure is blocking #41436.

**Comment by Moisan** [comment]
I was trying to bump the version of `mapserver` to see if it would smooth things out for #41436.

**Comment by alebcay** [comment]
Yes, sadly there's still something pretty broken with it.

**Comment by SMillerDev** [comment]
@brewtestbot test this please!

**Comment by MindTooth** [comment]
https://github.com/mapserver/mapserver/releases/tag/rel-7-4-2

**Comment by chenrui333** [comment]
@BrewTestBot test this please

**Comment by Moisan** [comment]
I bumped the version to `7.4.2`. I also spotted that upstream uses `swig` version 3, so I changed the dependency accordingly and it fixes the python code generation errors. The cmake script tries to install in `lib` instead of the typical `libexec`. Let's see how the CI respond to this 🤞 .

**Comment by chenrui333** [comment]
Any update on this PR?

File: Formula/mapserver.rb [hunk 1]
```diff
@@ -75,6 +68,6 @@ def install
 
   test do
     assert_match version.to_s, shell_output("#{bin}/mapserv -v")
-    system "python2.7", "-c", "import mapscript"
+    system "python", "-c", "import mapscript"
```
Comment (Bo98) [inline Formula/mapserver.rb]: Should this not be `python3`?

**Comment by Bo98** [comment]
Let's revisit this after Python 3.8 (#45337).

**Comment by thiscodecc** [comment]
No formula or cask found for "php5".

**Comment by SMillerDev** [comment]
@BrewTestBot test this please

**Comment by fxcoudert** [comment]
```
mapserver:
  * easy-install.pth files were found
    These .pth files are likely to cause link conflicts. Please invoke
    setup.py using Language::Python.setup_install_args.
    The offending files are:
      /usr/local/opt/mapserver/lib/python3.7/site-packages
```

**Review by chenrui333 — APPROVED** [review]

**Pull request closed without merging**